reports/Oregon.md

```diff
@@ -677,6 +691,158 @@ id: or-portland-45
 * https://twitter.com/PredatorFiles/status/1278635513746870276
 
 
+### Police fire on gathered protestors from within Justice Center | July 3rd
+
+Prior to the incident below, where police charged protestors, police inside the Justic Center fired pepper balls at gathered protestors.
```

Comment:
this could become incorrect if new incidents are inserted in between.  Maybe the two should just be merged into one entry?